alpine/rc.conf

```diff
@@ -0,0 +1,241 @@
+# Global OpenRC configuration settings
```

Comment:
It makes sense for this file to only contain the setting we set. I had to diff it with the default one getting installed to find out which extra options got enabled